mne/time_frequency/spectrum.py

```diff
@@ -429,7 +429,7 @@ def _check_values(self):
         assert len(self._dims) == self._data.ndim, (self._dims, self._data.ndim)
         assert self._data.shape == self._shape
         # negative values OK if the spectrum is really fourier coefficients
-        if np.iscomplexobj(self._data):
+        if "taper" in self._dims:
```

Comment:
reverting to reduce churn, esp. given that we now plan to deprecate support for complex data anyway.